skietbaan_fe/src/bootstrap/NavbarMenuStyle.css

```diff
@@ -56,47 +66,60 @@
   }
 
   .columns-v2{
-    width: 12%;
+    width: 25%;
   }
 
   .columns{
-    width: 12%;
+    width: 20%;
   }
 
   .first-row-navbar{
-    height: 68px;
+    height: 56px;
     padding-top: 15px;
     padding-bottom: 15px;
     width: 100%;
   }
   
   .navbar-admin{
-    background-color: #F3F4F9;
-    width: 100%;
+    background-color: #000000;
+    width: 66%;
     text-align: center;
+    margin-left: 17%;
+    top: 0;
+    z-index: 100;
+    position: fixed;
+    border-bottom-left-radius: 20px;
+    border-bottom-right-radius: 20px;
+    border-top-left-radius: 0px;
+    border-top-right-radius: 0px;
+  }
+
+  .nav-content{
+    text-align: center;
+    background-color: #000000;
   }
 }
 
-@media only screen and (max-width:579px){
+@media only screen and (max-width:575px){
   .dropdown-menu.show{
     background-color: #F3F4F9;
     border: none;
     border: 1px solid  #F3F4F9;
   }
 
   .leaderboard-icon-grey{
```

Comment:
`.leaderboard-icon-grey` and 
`.add-score-icon-grey` and 
`.create-icon-grey` are the same

Reply:
okay made a generic one